src/main/java/core/basesyntax/Lottery.java

```diff
@@ -0,0 +1,13 @@
+package core.basesyntax;
+
+import java.util.Random;
+
+public class Lottery {
+    public Ball getRandomBall() {
+        ColorSupplier randomColor = new ColorSupplier();
```

Comment:
this is not a color yet
```suggestion
        ColorSupplier colorSupplier = new ColorSupplier();
```